pkgs/tools/networking/nss-mdns/default.nix

```
@@ -22,6 +22,8 @@ stdenv.mkDerivation rec {
     "--enable-avahi"
     # Connect to the daemon at `/var/run/avahi-daemon/socket'.
     "--localstatedir=/var"
+    # Enable configuration through `/etc/mdns.allow`.
```

```suggestion
    # Read configuration at `/etc/mdns.allow`, not ...
```

...and add where it'd look otherwise

---

Pushed a fixup addressing this. I'll squash if there's no further changes requested.